Tools/PIDML/README.md

```diff
@@ -31,10 +36,10 @@ This is a wrapper around PidONNXModel that contains several models. It has the p
 
 Then, obligatory parameters for the interface:
 - a vector of int output PIDs
-- a 2-dimensional LabeledArray of *p*T limits for each PID, for each detector configuration. It describes the minimum *p*T values at which each next detector should be included for predicting given PID
+- a vector of *p*T cuts for each PID.
 - a vector of minimum certainties for each PID for accepting a track to be of this PID
 - boolean flag: whether to switch on auto mode. If true, then *p*T limits and minimum certainties can be passed as an empty array and an empty vector, and the interface will fill them with default configuration:
-  - *p*T limits: same values for all PIDs: 0.0 (TPC), 0.5 (TPC + TOF), 0.8 (TPC + TOF + TRD)
+  - *p*T limits: 0.5 for all PIDs
```

Comment:
I would still keep 0.0, 0.5 and 0.8 for all the 3 configs (see my comment on PidOnnxInterface)